src/index.html

```diff
@@ -9,10 +9,58 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="styles/style.scss"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonimous"
```

Comment:
There is a typo in the `crossorigin` attribute value. It should be 'anonymous' instead of 'anonimous'.